packages/expo-face-detector/ios/EXFaceDetector/EXFaceEncoder.h

```diff
@@ -14,6 +14,7 @@
 @interface EXFaceEncoder : NSObject
 
 - (instancetype)initWithTransform:(CGAffineTransform)transform;
+- (instancetype)initWithRotationTransform:(EXFaceDetectionAngleTransformBlock)transform;
```

Comment:
Maybe place it as last one 🤔 I mean after `(instancetype)initWithTransform:(CGAffineTransform)transform withRotationTransform:(EXFaceDetectionAngleTransformBlock)rotationTransform;`

Reply:
Why? My approach is to begin with most general use case, and go down to more specific ones.